### DISCUSSION TITLE
Welcome to GRASS Discussions!

### DISCUSSION THREAD

**Discussion by NikosAlexandris** [OSGeo/grass#1841] [General]
## 👋 Hello! 💚

📢 Following on #1826, [GitHub Discussions](https://github.com/OSGeo/grass/discussions) is put on trial as a place to connect with other members of the GRASS community. You are welcome to share the [Discussions](https://github.com/OSGeo/grass/discussions) link 🔗.

> Discussions is not a _replacement for_ but an _addition to_ the forums for [developers](https://discourse.osgeo.org/c/grass/developer/61) and [users](https://discourse.osgeo.org/c/grass/grass-user/70), but it provides an additional way to ask questions and discuss right on GitHub.

---

**You are warmly welcome to**

- Ask questions ☝️  
  > [There is no such thing as a dumb question.](https://www.goodreads.com/quotes/538156-there-are-naive-questions-tedious-questions-ill-phrased-questions-questions-put)

  — Carl Sagan

- Share your curious ideas 🤔 💡
  > [Be less curious about people and more curious about ideas.](https://en.wikiquote.org/wiki/Marie_Curie)

  — Marie Curie

- Engage with other members 🤝 💚 

---

**Remember**,

- Submit bug reports and/or feature requests at https://github.com/OSGeo/grass/issues 🐛 

- We Are The Community 👐
  > We are respectful toward others in accordance with our [Code of Conduct](https://github.com/OSGeo/grass/blob/main/CODE_OF_CONDUCT.md).

---

Feel free to introduce yourself and share what you do with GRASS 💚

**Comment by NikosAlexandris** [comment]
Great start in the _Discussions_ section in the last 3 days!

I would like to remind all of us (@CarlosGrohmann @veroandreo @YannChemin), though **it is by no means a _must_**, that we can make our posts more readable with simple Markdown syntax (please see [www.markdownguide.org](https://www.markdownguide.org/)) or simply using the basic text formatting buttons available in the header of the text editing box.

For example, enclosing a filename or a module name in backticks `` ` `` is easy and will make it easier to read through a message (e.e. `i.landsat.import`).
Thank you, Nikos

ps- I am yet to try sending a normal e-mail with Markdown syntax as a reply/comment in a Discussion and see whether the syntax is considered or not.

**Reply by wenzeslaus** [reply]
About the names and backticks, it is also a common paractice to make names emphasized/cursive and use backticks for pieces of code as in "note usage of _ls_ in `ls -la`".

**Comment by ZenithFlux** [comment]


**Reply by CarlosGrohmann** [reply]
@ZenithFlux, wouldn't this be a new discussion topic? I feel it won't get the necessary attention here, as a reply to Nikos post

**Reply by NikosAlexandris** [reply]
(If I could, I would remove this post, as there is already one in the corresponding issue. Please @ZenithFlux consider removing it.)

**Reply by NikosAlexandris** [reply]
Thank you @ZenithFlux
